README.md

```diff
@@ -12,6 +12,67 @@ Here are some links to know more:
 
 The APIs are documented in [API.md](./API.md).
 
+## Develop and running the server locally
+### Install the local tooling
+You will need a recent enough version of [Yarn](http://yarnpkg.com/),
+version 1.10 is known to work correctly.
+You can install it into your home directory on Linux and probably OS X with:
+
+```bash
```

Comment:
Maybe just link to: https://yarnpkg.com/getting-started/install

Reply:
I just copied this from the profiler's README; but I'm not opposed.

Reply:
So yarn's website has new instructions for yarn v2, this is a bit complex.
I added a line pointing to yarn v1's install insturctions.